uber/templates/registration/index_base.html

```diff
@@ -11,11 +11,29 @@
 </script>
     {% include "check_in.html" %}
 
+  {% if c.REG_USES_BARCODES %}
+    {% set detect_barcode_scan = True %}
+    <script type="text/javascript">
+        var setBarcodeField = function () {
+            if ($('input[name=badge_num]').length === 1) {
```

Comment:
I wasn't sure if we should force focus on the search bar here. The search bar is already pre-focused on page load (some JS a bit above this) so that should cover our bases as far as the workflow goes, and I was a little concerned about not 'respecting' the field the admin has focused. I'm willing to add that in if it's important to you, though.